src/entities/donation.ts

```diff
@@ -128,12 +128,8 @@ export class Donation extends BaseEntity {
 
   // these are all the donations before monoswap was updated
```

Comment:
Remove this comment haha this was when I first did the script. So we don't get confused later.

Reply:
Or update it to something more related to the current code.

Reply:
Thanks , I deleted the outdated comment